maestrowf/datastructures/core/executiongraph.py

```diff
@@ -14,7 +14,7 @@
 from maestrowf.datastructures.dag import DAG
 from maestrowf.datastructures.environment import Variable
 from maestrowf.interfaces import ScriptAdapterFactory
-from maestrowf.utils import create_parentdir, get_duration
+from maestrowf.utils import create_parentdir, get_duration, round_datetime_seconds
```

Comment:
It looks to me this is the only style error. Otherwise, could you put the stackoverflow link in the comments? You can split the link between two lines to fit in the character limit.